src/DevToys.Tools/Tools/EncodersDecoders/JsonWebToken/JsonWebTokenDecoderGuiTool.cs

```diff
@@ -133,14 +140,21 @@ public IUIStack ViewStack
                         .OnToggle(OnValidateTokenChanged)
                 )
                 .WithChildren(
-                    Setting("jwt-decode-validate-token-issuer-signing-key-setting")
+                    SettingGroup("jwt-decode-validate-token-issuer-signing-key-setting")
                         .Icon("FluentSystemIcons", '\ue30a')
                         .Title(JsonWebTokenEncoderDecoder.DecodeValidateTokenIssuerSigningKeyTitle)
                         .InteractiveElement(
                             _validateTokenIssuerSigningKeySwitch
                                 .OnText(JsonWebTokenEncoderDecoder.Yes)
                                 .OffText(JsonWebTokenEncoderDecoder.No)
                                 .OnToggle(OnValidateTokenIssuerSigningKey)
+                        )
+                        .WithChildren(
+                            _isSignatureKeyInBase64FormatSwitch
```

Comment:
Hello,
Thanks for creating this pull request. :) It looks great to me in overall!
Would it be possible to use a `IUISetting` with a boolean setting here instead of a `IUISwitch`? We could use a title and description on the setting to indicate what this option is doing. Also, I think using a `IUISetting` will magically fix the misalignment of the control vertically.

Reply:
![image](https://github.com/DevToys-app/DevToys.Tools/assets/40485241/039012a3-835d-4de1-b4f9-aa9434cb9bf4)
Like so?

Another approach could be to provide a switch configuration `Action<IUISwitch>` in the `IUISetting` handle method, which would allow setting the On/Off text.
```cs
public static IUISetting Handle(
    this IUISetting element,
    ISettingsProvider settingsProvider,
    SettingDefinition<bool> settingDefinition,
    string? stateDescriptionWhenOn,
    string? stateDescriptionWhenOff,
    Action<bool>? onToggled,
-----------------------------------
    Action<IUISwitch>? switchConfigure 
-----------------------------------
)
```

Reply:
Yes! Like this! 😁
The `Action<bool>? onToggled` will take care of setting/reading the value of your `SettingDefinition<bool>` automatically.